app/workflows/same_appeal_substitution_tasks_factory.rb

```
@@ -20,11 +22,63 @@ def create_substitute_tasks!
   def create_tasks_for_distributed_appeal
     if @appeal.docket_type == Constants.AMA_DOCKETS.hearing && selected_tasks_include_hearing_tasks?
       send_hearing_appeal_back_to_distribution
+    elsif evidence_submission_task_selected?
+      resume_evidence_submission
     elsif no_tasks_selected?
       reopen_decision_tasks
     end
   end
 
+  def evidence_submission_task_selected?
+    selected_tasks = Task.where(id: @task_ids[:selected]).order(:id)
+    !selected_tasks.of_type(:EvidenceSubmissionWindowTask).empty?
+  end
+
+  ATTRIBUTES_EXCLUDED_FROM_TASK_COPY = %w[id created_at updated_at
+                                          status closed_at placed_on_hold_at].freeze
+
+  def copy_esw_task_with_ancestors(task)
+    parent = task.parent
+    return unless parent
+
+    new_task_attributes = task
+      .attributes
+      .except(*ATTRIBUTES_EXCLUDED_FROM_TASK_COPY, ["parent_id"])
+
+    existing_new_parent = @appeal.reload.tasks.open.find { |option| option.type == parent.type }
+    new_parent = existing_new_parent || copy_esw_task_with_ancestors(parent)
+
+    return unless new_parent
+
+    new_task_attributes["parent_id"] = new_parent.id
+    Task.create!(new_task_attributes)
+  end
+
+  def resume_evidence_submission
+    esw_task = @appeal.tasks.of_type(:EvidenceSubmissionWindowTask).closed.order(:id).last
+    esw_task_params = @task_params[esw_task.id.to_s]
+    unless esw_task_params["hold_end_date"]
+      fail "Expecting hold_end_date creation parameter for EvidenceSubmissionWindowTask from #{esw_task.id}"
+    end
+
+    evidence_submission_hold_end_date = Time.find_zone("UTC").parse(esw_task_params["hold_end_date"])
```

What happens if `esw_task_params["hold_end_date"]` is an unparsable string? Is the result acceptable?

---

In the current implementation, the evidence submission window task would be created with a 90 day window.  I think we should throw an error if `esw_task_params["hold_end_date"]` is an unparsable string, so I added error handling.  Good catch!